### ISSUE TITLE
Trait method cfgs aren't honored

### ISSUE THREAD

**Issue by TheDan64** [audunhalland/entrait#18]
Suppose I have something like:

```rust
#[cfg(feature = "baz")]
struct Baz;

#[entrait]
trait Foo {
    fn bar();
    #[cfg(feature = "baz")]
    fn baz() -> Baz;
}
```

You'll get an error saying Baz doesn't exist when you don't provide the baz feature flag - entrait doesn't seem to propagate cfgs to the output code.

**Comment by audunhalland** [comment]
This is indeed a bug, working on a fix.

**Comment by audunhalland** [comment]
released 0.5.1 containing a fix for this.
